pmm/pmm2-new-version.groovy

```diff
@@ -0,0 +1,92 @@
+pipeline {
+    agent {
+        label 'master'
+    }
+    parameters {
+        string(
+            defaultValue: '2.0.0',
```

Comment:
I think it is safer to remove the default value.

Reply:
ok